website/docs/r/repository.html.markdown

```diff
@@ -116,6 +116,8 @@ initial repository creation and create the target branch inside of the repositor
 
 * `topics` - (Optional) The list of topics of the repository.
 
+~> Note: This attribute is not compatible with the ``github_repository_topics`` resource. Use one of them
```

Comment:
Can you please add a note here that describes when you would want to use one resource vs. the other?

Reply:
Done! I've expanded the docs a bit, and linked to #1845, which shows how I will use it!